pallets/gear/src/tests.rs

```diff
@@ -3896,6 +3896,24 @@ fn test_reply_to_terminated_program() {
     })
 }
 
+#[test]
```

Comment:
will you introduce the test from audit repo?

Reply:
Done [here](https://github.com/gear-tech/gear/pull/1276/commits/99850daf49da36bb9c95ec63781a3d9560132e95)